src/structures/GuildChannel.js

```diff
@@ -539,6 +539,17 @@ class GuildChannel extends Channel {
     return this.permissionsFor(this.client.user).has(Permissions.FLAGS.MANAGE_CHANNELS);
   }
 
+  /**
+ * Whether the channel is manageable by the client user
+ * @type {boolean}
+ * @readonly
+ */
+  get manageable() {
+    if (this.client.user.id === this.guild.ownerID) return true;
+    return this.permissionsFor(this.client.user).has([Permissions.FLAGS.MANAGE_CHANNELS,
```

Comment:
permissionsFor()'s return is a nullable (in this case if the client user's member isn't cached)

```js
    const permissions = this.permissionsFor(this.client.user);
    return permissions && permissions.has([Permissions.FLAGS.MANAGE_CHANNELS, Permissions.FLAGS.VIEW_CHANNEL]);
```

Reply:
Good catch. This also fixes the max-len issue from before so 2 birds with 1 stone.

Reply:
The permissions check should be cast to a boolean to ensure return type compatibility.